### PR TITLE
Added a simple templates project to make editing easier.

### PR DESCRIPTION
**Pull request by garethj-msft**
Simple non-building project added to the solution to make it easier to comprehend and edit templates.  Also couple of line endings changes that git imposes for Windows auto-line endings.

### CODE REVIEW

File: GraphODataTemplateWriter.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
```
Comment (peternied) [inline GraphODataTemplateWriter.sln]: Please do not update the Visual Studio version for this sln

File: LICENSE [hunk 2]
```diff
@@ -1,25 +1,25 @@
-ODataGraphTemplateWriter
```
Comment (peternied) [inline LICENSE]: Looks like line endings were modified, please revert

**Comment by peternied** [comment]
Could you update the readme with how the usage should be changed?

File: GraphODataTemplateWriter.sln (hunk 1 above)
Comment (garethj-msft) [inline GraphODataTemplateWriter.sln]: per email discussion, this seems like the right change.

**Comment by garethj-msft** [comment]
I updated the readme and removed the line ending-only changes per comments.

File: LICENSE (hunk 2 above)
Comment (garethj-msft) [inline LICENSE]: Reverted

**Pull request merged**